packages/hash/integration/src/tests/model/ontology/entity-type.model.test.ts

```diff
@@ -123,6 +148,18 @@ beforeAll(async () => {
         },
         ordered: false,
       },
+      [previousAddressLinkTypeModel.schema.$id]: {
+        type: "array",
+        items: {
+          oneOf: [
+            // When adding links in entity type definitions the `$ref` is
```

Comment:
I'm usually for adding comments wherever because more information is usually better. I don't really see the purpose of writing this though and it actually feels a little off to include it. I don't necessarily feel strongly about removing it, I just don't want to see us explaining things over and over, inconsistently, in the wrong places.

I don't think we should be explaining the syntax everywhere we use it, I think people should look at the types, or indeed the type system docs for the reasoning.

Reply:
Apologies, didn't mean for this to get committed - it's a direct copy of the comment a few lines up for the other outgoing link type (not introduced in this PR). Would you like me to remove both comments?

Reply:
Probably? Still not super bothered about having the comment here, it just feels quite strange

Reply:
Have removed them in https://github.com/hashintel/hash/pull/1115/commits/327315a0e276065027f30b11e8659d73f2bbaf83, I don't think they're very useful either and add quite a lot of lines in the file.